src/riscv-hybrid-integration.adoc

```diff
@@ -279,9 +279,20 @@ shown in xref:default-csrnames-added[xrefstyle=short].
 
 {cheri_default_ext_name} optionally allows <<MODESW>> to execute in debug mode.
 
-When entering debug mode, the core always enters {cheri_cap_mode_name}.
+When entering debug mode, whether the core enters {cheri_int_mode_name} or
+{cheri_cap_mode_name} is UNSPECIFIED.
 Implementations may optionally support switching CHERI execution mode by
-executing the <<MODESW>> from the program buffer.
+executing <<MODESW>> from the program buffer.
+
+The current mode can be read from <<dinfc>>.
```

Comment:
Note that we currently have not described what happens when reading `dinfc` from Integer Pointer Mode -- I just noticed this!

Reply:
we do - see above